_posts/2019-01-17-image-service-deprecation.md

```diff
@@ -10,9 +10,9 @@ categories:
 tags: []
 ---
 
-The AGRC brown out of the mapserv image services was very successful. We want to thank you for your participation and apologize for any inconveniences. The few people that were affected and caught by surprise are now informed and on the migration path forward.
+The AGRC brownout of the mapserv image services was very successful. We want to thank you for your participation and apologize for any inconveniences. The few people that were affected and caught by surprise are now informed and are on the migration path forward.
 
-With that behind us, AGRC has chosen the date of **Thursday, February 14** to permanently remove the image services. The deprecation Utah Change Management System reference number is `CHG0019495` and will be discussed at the next two Change Management meetings.
+With the brownout behind us, AGRC has chosen the date of **Thursday, February 14**, to permanently remove the image services. The deprecation Utah Change Management System reference number is `CHG0019495` and will be discussed at the next two change management meetings.
```

Comment:
I'm thinking maybe somewhere too, we should also just briefly mention how people should move forward/prepare for this change? Or should everyone already be ready since they participated in the test brownout? Either way, it might be good to know exactly what might happen when the brownout occurs, you know? Like how should people prepare/what should they expect exactly?

Reply:
Yes, I would like to work on the language for the contact include template. It is a little complicated because it is generic and used throughout the entire website. Some contacts don't have certain contact information parts.